.vscode/scripts/import-example-app.sh

```diff
@@ -27,20 +27,8 @@ else
   rm -rf app/
   cp -a $VELOCITAS_EXAMPLES_PATH/$CHOSEN_EXAMPLE/. app/
 
-  pip-compile ./app/requirements.txt
-  pip-sync ./app/requirements.txt
-
-  if [[ -f "./app/requirements.txt" ]]; then
-    pip install -r ./app/requirements.txt
-  fi
-
-  if [[ -f "./app/requirements-links.txt" ]]; then
-    pip install -r ./app/requirements-links.txt
-  fi
-
-  if [[ -f "./app/tests/requirements.txt" ]]; then
-    pip install -r ./app/tests/requirements.txt
-  fi
+  pip-compile -r ./requirements.in
```

Comment:
IMO it make sense to hide the output
```suggestion
  pip-compile -r -q ./requirements.in
```